### ISSUE TITLE
Wal mode

### ISSUE THREAD

**Issue by rjwillson** [DMS-Aus/Roam#362]
I just found a way to turn off disable Write Ahead Logging (WAL) mode with SpatiaLite/sqlite and GeoPackage files when using QGIS

But when Roam opens the QGIS project I see that it turns wal on and now I can see that Roam re-enables the wal mode as shown in the screen clip below

Two questions: (1) does the wal mode improve performance in Roam when collected data into SpatiaLite and GeoPackage files; and (2) depending on answer to question 1 is it possible to somehow disable it if performance gain is happens to be minimal?

I have set Roam up to work with Dropbox and all the additional -wal and -shm files confuse the end users when they all start syncing and sometimes creating conflicted copies

![image](https://cloud.githubusercontent.com/assets/11098144/23313049/0c9a5ad2-fa8a-11e6-83ff-55c24e5bbba0.png)

**Comment by NathanW2** [comment]
@rjwillson yeah I'm not 100% sure why I added this I suspect it was to fix something that doesn't happen anymore.  I'm happy to get a pull request to remove or at least disable it for now.
